src/dftbp/dftbplus/parser.F90

```diff
@@ -3925,7 +3922,13 @@ subroutine readOptions(node, ctrl)
     if (ctrl%tReadChrg) then
       call getChildValue(node, "ReadChargesAsText", ctrl%tReadChrgAscii, .false.)
     end if
-    call getChildValue(node, "WriteChargesAsText", ctrl%tWriteChrgAscii, .false.)
+
+    call getChildValue(node, "WriteCharges", ctrl%tWriteCharges, .true.)
+    call getChildValue(node, "WriteChargesAsText", ctrl%tWriteChrgAscii, .false., child=child)
```

Comment:
Should only need to check for this flag (and the next if block) if tWriteCharges == T